api/base/settings/__init__.py

```diff
@@ -28,14 +26,14 @@
         assert getattr(local, setting, None) and getattr(local, setting, None) != getattr(defaults, setting, None), '{} must be specified in local.py when DEV_MODE is False'.format(setting)
 
 def load_origins_whitelist():
-    global ORIGINS_WHITELIST
+    global CORS_ORIGIN_WHITELIST
```

Comment:
Since we ended up keeping the `CORS_ORIGIN_WHITELIST` in api settings/admin settings, we don't want to rename it the global `ORIGINS_WHITELIST` to the same name  in order avoid confusion.